scripts-dev/build_debian_packages.py

```diff
@@ -30,7 +30,6 @@
     "debian:bullseye",  # (EOL ~2024-07) (our EOL forced by Python 3.9 is 2025-10-05)
     "debian:bookworm",  # (EOL not specified yet) (our EOL forced by Python 3.11 is 2027-10-24)
```

Comment:
bookworm has since [had an EOL specified](https://wiki.debian.org/DebianReleases#:~:text=packages%20for%20stable.-,Production%20Releases,-Version):

```suggestion
    "debian:bookworm",  # (EOL 2026-06) (our EOL forced by Python 3.11 is 2027-10-24)
```